Rakefile

```diff
@@ -0,0 +1,14 @@
+require 'html-proofer'
+
+task :test do
+  sh "bundle exec jekyll build"
```

Comment:
Maybe make this as a build task and then make test depends on it.

Reply:
You already have a build task that is `bundle exec jekyll build`
If for some reason we need to add more steps, then it would make sense to make a build task, but it won't make sense to depend test task on said build task

Reply:
If you have a build task, its clear for everyone how to build the project. In this case you could make netlify use `rake build` and if anything changes you can always change just the Rakefile instead of netlify configuration. Same think to Travis. Besides, I prefer to write `rake build` than `bundle exec jekyll build`. Also a develop task would be awesome (`bundle exec jekyll serve`).

Reply:
Still not seeing why test should depend on build task. Also this project is in jekyll so it is expected to use that command rather than anything else. Even so the build and develop tasks are not the scope of this PR.

Reply:
It should depend on a build task because in order to test you need to build it first.